### DISCUSSION TITLE
Beş köy sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#223] [Zeka Sorusu] [answered]
Bir şehirde 5 köy vardır. Bu köyler Güzel Köy, Şirin Köy, Kuzulu Köy, Yeşil Köy ve Çukur Köy olarak adlandırılmaktadır. Köyler şehrin dağlık, ormanlık, deniz kıyısı, çöl ve vadi bölgelerindedir. Köylerin kar yağışları 13cm, 28cm, 33cm, 45cm ve 55cm olarak ölçülmüştür. Aşağıdakiler de bilinmektedir.
• Çöldeki köy en az kar yağışını, ormandaki köy ise en çok kar yağışını almıştır.
• Kuzulu köy şehrin dağlık bölgesindedir.
• Güzel Köy, Yeşil Köy’e göre daha fazla kar yağışı almıştır. • Şirin Köy, Çukur Köy’e göre daha fazla, Kuzulu Köy’e göre daha az yağış almıştır.
• Dağlık bölgedeki köy 33cm kar yağışı almışken deniz kıyısındaki köy 28cm kar yağışı almıştır.
Soru 1: En fazla kar yağışını hangi köy almıştır?
Soru 2: Şirin köy kaç cm yağış almıştır?
Soru 3: Yeşil köy şehrin hangi bölgesindedir?

**Comment by FatihOnder81** [comment, marked as answer]
Orman 55 Güzel köy
Vadİ     45 Yeşil köy
Dağlık  33 Kuzulu köy
Deniz   28 Şirin köy
Çöl       13 Çukur köy

1-Güzel köy
2-28cm
3-Vadi
